pkg/auth/dependency/oauth/handler/response_mode.go

```diff
@@ -0,0 +1,105 @@
+package handler
+
+import (
+	"fmt"
+	"html/template"
+	"net/http"
+	"net/url"
+
+	coreurl "github.com/skygeario/skygear-server/pkg/core/url"
+)
+
+const htmlRedirectTemplateString = `<!DOCTYPE html>
+<html>
+<head>
+<meta http-equiv="refresh" content="0;url={{ .redirect_uri }}" />
+</head>
+<body>
+<script>
+window.location.href = "{{ .redirect_uri }}"
+</script>
+</body>
+</html>
+`
+
+const formPostTemplateString = `<!DOCTYPE html>
+<html>
+<head>
+<title>Submit this form</title>
```

Comment:
For better UX, we can use localized app name?

Reply:
90% people wont see this. When they see this, what they need to do is to submit this form. But localizing this string has some overhead.

Reply:
I think at least this message should be moved to body (and may need localization in future), since users seldom pay attention to title.

Reply:
Move the message as `<noscript>` to `<body>`